openedx/stanford/cms/djangoapps/contentstore/tests/test_captions.py

```diff
@@ -56,7 +56,6 @@ def setUp(self):
 
         # Add video module
         data = {
-            # 'parent_locator': self.location.to_deprecated_string(),
```

Comment:
All other changes are just in our forked files;
edx has removed their copies upstream.